api/api/clubs/models.py

```diff
@@ -17,3 +21,22 @@ class Club (models.Model):
     description = models.TextField()
     email = models.EmailField(max_length=50, unique=True)
     logo = models.URLField(max_length=500)
+
+
+class ClubRegistrationRequests(models.Model):
+    class RegistrationStatus(models.TextChoices):
+        PENDING = 'Pending', _('Pending')
+        APPROVED = 'Approved', _('Approved')
+        REJECTED = 'Rejected', _('Rejected')
+
+    user = models.ForeignKey(User, related_name='registered_clubs', on_delete=models.CASCADE)
+    club = models.ForeignKey(Club, related_name='registration_requests', on_delete=models.CASCADE)
+    fee_submitted = models.BooleanField(default=False)
+    status = models.CharField(max_length=10, choices=RegistrationStatus.choices,
+                              default=RegistrationStatus.PENDING)
+    remark = models.TextField(blank=True)
+    updated_by = models.ForeignKey(
+        User, related_name='updated_club_registration_requests',
```

Comment:
too big related name?

Reply:
Tried to give a meaningful name . May be changed to `updated_registration_requests` or `updated_requests` ? But they lack context I think . Any suggestion ?

Reply:
`last_updated_requests`